students/wachulec_maria/lesson_01_basics/hello_harry.py

```diff
@@ -0,0 +1,2 @@
+name = input("What is your name?: ")
+print("Hello, " + name + '.')
```

Comment:
There should be an exclamation mark at the end.
This code does not pass any of tests on snakify website https://snakify.org/lessons/print_input_numbers/problems/hello_harry/
It's a good idea to run tests before submitting code.